apps/src/applab/Visualization.jsx

```diff
@@ -20,6 +20,16 @@ var styles = {
   },
   phoneFrame: {
     marginBottom: 0
+  },
+  screenBlock: {
+    backgroundColor: 'rgba(0, 0, 0, 0.05)',
```

Comment:
Hah! I actually looked at using white instead of black again, and we decided it does look better :-P